packages/next-codemod/lib/utils.ts

```diff
@@ -38,71 +38,82 @@ export function onCancel() {
 export const TRANSFORMER_INQUIRER_CHOICES = [
   {
     title:
-      'name-default-component: Transforms anonymous components into named components to make sure they work with Fast Refresh',
-    value: 'name-default-component',
+      'Transforms the deprecated automatically injected url property on top level pages to using withRouter',
```

Comment:
let's keep all predicates simple
```suggestion
      'Transform the deprecated automatically injected url property on top level pages to using withRouter',
```